content/news/preview-20190606.md

```diff
@@ -0,0 +1,59 @@
+It has been quite a while since our last release, but not to worry, we are still going - mostly rewriting tons of code not visible to the outside eye. While not presenting a full playtest we want to take the time to introduce the community to some upcoming changes to look forward to.
+
+### Singleplayer savegames
+
+Probably the most wanted missing feature, the next version will include support for savegames in both singleplayer and the campaign. No more losing all progress to a stray dog eating your Tanya!
+
+<div style="text-align:center" markdown="1">
+<img src="/images/news/20190606-savegame-d2k.png" alt="The savegame load screen in D2k." width="600px">
+
+Longer missions can now be played in several takes instead of needing you to pull through in one go.
+</div>
+
+### Overhauled UI
+
+Our spectator UI has always been noted to be clunky and taking too much space off the screen. With the next update we are happy to finally introduce a more slim and focused spectator UI.
```

Comment:
English is a bit weird here - how about `Our spectator UI has always been noted to be clunky and take too much space on the screen`?